app/experimenter/experiments/tests/test_models/test_models_nimbus.py

```diff
@@ -542,34 +539,6 @@ def test_computed_enrollment_days_returns_fallback(self):
             experiment.proposed_enrollment,
         )
 
-    @parameterized.expand(
-        [
-            (NimbusExperimentFactory.Lifecycles.PAUSING_REVIEW_REQUESTED,),
-            (NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE,),
-            (NimbusExperimentFactory.Lifecycles.PAUSING_APPROVE_WAITING,),
-        ]
-    )
-    def test_computed_enrollment_days_returns_fallback_while_pause_pending_approval(
-        self, lifecycle
-    ):
-        expected_days = 99
-        experiment = NimbusExperimentFactory.create_with_lifecycle(
-            lifecycle,
-            proposed_enrollment=expected_days,
-        )
-        # Set the span to 5 days, but that shouldn't apply while pending approval
-        experiment.changes.filter(
-            old_status=NimbusExperiment.Status.DRAFT,
-            new_status=NimbusExperiment.Status.LIVE,
-        ).update(changed_on=datetime.datetime.now() - datetime.timedelta(days=5))
-        experiment.changes.filter(experiment_data__is_paused=True).update(
-            changed_on=datetime.datetime.now()
-        )
-        self.assertEqual(
-            experiment.computed_enrollment_days,
-            expected_days,
-        )
-
```

Comment:
And all this stuff

Reply:
Where the heck did this come from

Reply:
Just landed in main after I closed my PR

Reply:
If you're using VSCode it has a thing where you can revert individual blocks of changes within files

Reply:
Oh I get it.